docs/content/en/install/local/_index.md

```diff
@@ -1,441 +1,2316 @@
+helm install cert-manager jetstack/cert-manager --namespace cert-manager \
+     --create-namespace --set installCRDs=true
+```
 
-You'll need the K8ssandra superuser name and password in order to access Cassandra utilities and do things like generate a Stargate access token.
+**Output:**
 
-To retrieve K8ssandra superuser credentials:
+```bash
+NAME: cert-manager
+LAST DEPLOYED: Thu Jan 27 15:28:59 2022
+NAMESPACE: cert-manager
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+cert-manager v1.7.0 has been deployed successfully!
+```
 
-1. Retrieve the K8ssandra superuser name:
+Then enter:
 
-    ```bash
-    kubectl get secret k8ssandra-superuser -o jsonpath="{.data.username}" | base64 --decode ; echo
-    ```
+```bash
+kubectx kind-k8ssandra-1
 
-    **Output**:
+helm install cert-manager jetstack/cert-manager --namespace cert-manager \
+     --create-namespace --set installCRDs=true
+```
 
-    ```bash
-    k8ssandra-superuser
-    ```
+**Output:**
 
-1. Retrieve the K8ssandra superuser password:
+```bash
+NAME: cert-manager
+LAST DEPLOYED: Thu Jan 27 15:28:59 2022
+NAMESPACE: cert-manager
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+cert-manager v1.7.0 has been deployed successfully!
+```
 
-    ```bash
-    kubectl get secret k8ssandra-superuser -o jsonpath="{.data.password}" | base64 --decode ; echo
-    ```
+Then enter:
 
-    **Output**:
+```bash
+kubectx kind-k8ssandra-2
 
-    ```bash
-    PGo8kROUgAJOa8vhjQrE49Lgruw7s32HCPyVvcfVmmACW8oUhfoO9A
-    ```
+helm install cert-manager jetstack/cert-manager --namespace cert-manager \
+     --create-namespace --set installCRDs=true
+```
 
-{{% alert title="Tip" color="success" %}}
-Save the superuser name and password for use in the [Quickstarts]({{< relref "/quickstarts" >}}), if you decide to follow those steps.
-{{% /alert %}}
+**Output:**
 
-## Next steps
+```bash
+NAME: cert-manager
+LAST DEPLOYED: Thu Jan 27 15:28:59 2022
+NAMESPACE: cert-manager
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+cert-manager v1.7.0 has been deployed successfully!
+```
 
-* If you're a developer, and you'd like to get started coding using CQL or Stargate, see the [Quickstart for developers]({{< relref "/quickstarts/developer" >}}).
-* If you're a Site Reliability Engineer, and you'd like to explore the K8ssandra administration environment including monitoring and maintenance utilities, see the [Quickstart for Site Reliability Engineers]({{< relref "/quickstarts/site-reliability-engineer" >}}).
+Then enter:
+
+```bash
+kubectx kind-k8ssandra-3
+
+helm install cert-manager jetstack/cert-manager --namespace cert-manager \
+     --create-namespace --set installCRDs=true
+```
+
+**Output:**
+
+```bash
+NAME: cert-manager
+LAST DEPLOYED: Thu Jan 27 15:28:59 2022
+NAMESPACE: cert-manager
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+cert-manager v1.7.0 has been deployed successfully!
+```
+
+### Install K8ssandra Operator in the control-plane
```

Comment:
We will probably want to preface this portion with instructions for how to add the K8ssandra helm repo -- this is no different than in 1.x -- we'll use a different chart in it, but it's the same repository so the helm command to add the repo would be the same.

Reply:
Ok, something like this? 

First, configure the K8ssandra Helm repository:

```console
helm repo add k8ssandra https://helm.k8ssandra.io/stable
```

Update your Helm repository cache:

```console
helm repo update
```

Verify that you see the `k8ssandra-operator` chart:

```console
helm search repo k8ssandra-operator
```